app/components/chat_form/chat_form.css

```diff
@@ -0,0 +1,9 @@
+.ChatForm {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.ChatForm > button {
```

Comment:
I’d use a stack to add spacing between the `textarea` and the `button` – but for such simple cases, I’m also fine with a simple descendent selector.